src/ucai/core/databricks.py

```diff
@@ -602,6 +602,22 @@ def _execute_uc_functions_with_serverless(
                 f"Failed to execute function with function name: {function_info.full_name}"
             ) from e
 
+    @override
+    def delete_function(
+        self,
+        full_function_name: str,
+        force: Optional[bool] = None,
+    ) -> None:
+        """
+        Delete a function by its full name.
+
+        Args:
+            full_function_name: The full name of the function to delete.
+                It should be in the format of "catalog.schema.function_name".
+            force: Force deletion even if the function is notempty. Defaults to None.
```

Comment:
what does None mean?

Reply:
```suggestion
            force: Force deletion even if the function is not empty. Defaults to None.
```

Reply:
Honestly I don't know :) The underlying sdk uses default value None, if it's none then it doesn't pass this field to the request body

Reply:
> The underlying sdk uses default value None

I think we can mention this if the underlying client decides what to do with None.